docs/versions/latest/Release.md

```diff
@@ -214,40 +220,51 @@ Special output is needed for the AsyncWorkflowItem.
 
 No special inputs are needed for the AsyncWorkflowItem.
 
-
 #### Example
 
 ```typescript
-import { Workflow, Out, In, Item, RootItem, AsyncWorkflowItem } from "vrotsc-annotations";
+import {
```

Comment:
I don't know what kind of auto formatter you are using, but these kind of changes everywhere are not needed. 

1. trailing commas don't make the code better
2. the line width you have set up should really be introduced.... we no longer have small screens for this wrapping to make sense.
3. Let's revert everything not related to your PR please

Reply:
Discussed with @VenelinBakalov , no need for reverting currently.

Reply:
@Michaelpalacce - we discussed that we can merge the current changes suggested by the autoformater (the formater should be handling some sonar/linting issues so might be a good approach to introduce) and next week we will check with Kalin and if something is not ok, we can align the settings and revert.
However, after that discussion a new issue popped up that might require a release this week. Because of that it would be best to either postpone this PR for next week or revert the changes and merge without them until we have covered the discussion with Kalin.

Reply:
The changes were reverted, so we can merge this one as well.